src/features/redux/bazar/bazarSlice.js

```diff
@@ -11,17 +11,20 @@ import {
   postCheckout,
 } from "./bazarThunkActions";
 
+const initialState = {
+  coupons: [],
+  categories: [],
+  products: [],
+  allProducts: [],
+  card: [],
```

Comment:
если тут имеется в виду Корзина, то неверный нейминг
```suggestion
  cart: [],
```